docs/blog/20/5-gettig_started_to_vitis_acceleration_flow_with_zynq_7000.md

```diff
@@ -0,0 +1,351 @@
+---
+date: 2020-12-07
+author: Anıl TIRLIOĞLU
+description: Getting Started to Vitis Acceleration Flow with Zynq 7000.
+axlang: "en"
+---
+
+# Getting Started to Vitis Acceleration Flow with Zynq 7000
+
+We all know, sometimes it is just hard to get start learning things from
+vendor documentations. This tutorial will follow beginner friendly steps to run
+your first accelerator on an FPGA.
+
+!!! Info
+    This tutorial will be more beneficial when used in
+    conjunction with the official Xilinx tutorial[^1f] published on github.
+
+There are two different device definitions in Xilinx. One is **embedded
+devices**[^2f],  other one is **data center accelerator cards**[^3f]. Embedded
```

Comment:
Buna benzer satırlarda 80 kuralı uygulanırken eğer otomatik yapıyorsan "**embedded" gibi kısımlarda "**" ifadesi yarıda kesildiği için kalınlaştırılması istenilen yerler aslında kalınlaşmıyor bazen. Bu gibi kısımlar düzeltilmeli.

Reply:
Bu benim de dikkatimi çekmişti. VSCode bu durumda editör içerisinde düzgün göstermiyor, haklısın. Fakat mkdocs ile siteyi derleyince düzgün gözüküyor. Bakabildiğim kadarıyla da Markdown speclerinde de izin verilen bir şey bu.

Github, VSCode gibi yerlerde düzgün gözükmesi için birleştirilebilir fakat sanıyorum bıraksak da çıkan sonuç düzgün olacak.

Reply:
Bilgi için teşekkürler.

Reply:
Ne demek :) markdownlint de hata vermeyince ona da güvendim biraz.